doc/langref.html.in

```diff
@@ -6926,6 +6926,27 @@ pub usingnamespace @cImport({
       {#syntax#}pub{#endsyntax#}. This can be used to forward declarations, giving precise control
       over what declarations a given file exposes.
       </p>
+      <p>
+      Another use case is splitting up the declaration of a data structure separately from its implementation:
+      </p>
+      {#syntax_block|zig|separate_impl.zig#}
+const Wine = struct {
+    name: [:0]const u8,
+    year: isize,
+    price: usize,
+    origin: Country,
+    pub usingnamespace _impl_Wine;
+};
+const _impl_Wine = opaque {
```

Comment:
The langref probably should not encourage underscore prefixed names?  This name should probably be CamelCase, too?  (`WineImpl` or `WinePrivate`?)